.github/workflows/gradle-reusable-test.yaml

```diff
@@ -112,18 +123,17 @@ jobs:
             sleep 1
           done
 
-      - name: Build
+      - name: Test log4j-samples-android
         id: build
         shell: bash
         env:
           LOG4J_VERSION: ${{ inputs.log4j-version }}
           LOG4J_REPOSITORY_URL: ${{ inputs.log4j-repository-url }}
         run: |
-          log4j-samples-android/gradlew -p log4j-samples-android \
-          --console plain \
```

Comment:
Note: `--console plain` is not necessary here. It's automatically switching to that mode in CI environment.